style.css

```diff
@@ -0,0 +1,229 @@
+/* media query when screen is larger than 768px. If [device width] is greater than or equal to the given pixels, run code */
+@media only screen and (min-width: 375px) and (max-width: 768px) {
+  @import url("https://fonts.cdnfonts.com/css/poppins");
+
+  body {
+    height: 100%;
+    background: #fff;
+  }
+
+  .logo img {
+    /* Logo Placeholder */
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: absolute;
+    height: 70px;
+    left: 12px;
+    top: calc(50% - 40px / 2);
+  }
+
+  .menu-icons {
+    /* Icons/Icon - Menu */
+
+    max-width: 100%;
+    height: 24px;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .menu-button {
+    /* Normal Button/Tertiary/Enabled */
+
+    /* Auto layout */
+
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 8px;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 24px;
+    top: calc(50% - 40px / 2);
+    border-radius: 8px;
+  }
+
+  .headline {
+    /* Headline */
+
+    /* Auto layout */
+    max-width: 100%;
+    display: flex;
+    position: relative;
+    flex-shrink: 0;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-top: 20px;
+    padding: 114px 24px;
```

Comment:
- [ ] The space between the header and headline is too large. Kindly reduce it to match the Figma

**Figma**
![scrn2](https://github.com/NoelLincoln/mportfolio/assets/99656687/6686a28f-eab5-4439-8b98-b70e2dacb146)

**Your design**
![scrn](https://github.com/NoelLincoln/mportfolio/assets/99656687/d2464125-0270-4575-ac39-071668523891)
.